themes/mlibrary_new/exhibit-builder/exhibits/gallery.php

```diff
@@ -0,0 +1,45 @@
+
+<section class="row">
+  <div class="col-xs-12 col-sm-9">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item"><?php echo link_to_exhibit(metadata('exhibit','title',array('no_escape' => true)));?></li>
+        <li class="breadcrumb-item active"><?php echo 'Exhibit Image'. metadata('exhibit_page','title',array('no_escape' => true)); ?></li>
+      </ol>
+  </div>
+  <div class="col-xs-12 col-sm-3">
+    <ul class="share-this">
+      <li>Share this Gallery:</li>
+      <li><a href="https://twitter.com/share" class="twitter-share-button" data-text="I just saw '<?php echo metadata('exhibit','title',array('no_escape' => true)); ?>' at the U-M Library Online Exhibits!" ><span class="sr-only">Tweet</span> </a></li>
+    </ul>
+   </div>
+</section>
+
+<div>
+<?php echo "Exhibit Image Gallery";
+echo "Browse all images in this exhibit by section and by the order they appear within each section.";
+echo "The University of Michigan Library has placed copies of these works online for educational and.";
+echo "research purposes. For more information about using images from this exhibit please visit the Rights Statement page.";?>
+</div>
+
+<?php set_exhibit_pages_for_loop_by_exhibit();
+
+foreach (loop('exhibit_page') as $exhibitsection) {
+$cards_in_subsection = '';
+$cards_in_section = '';
```

Comment:
Line 28 is unnecessary;  the variable is reassigned in line 31 before anything else can happen.

Reply:
Good catch, forget about it.